src/main/resources/db/migration/data/R__4_42__OFFENDER_NON_ASSOCIATIONS.sql

```diff
@@ -1,2 +1,2 @@
-INSERT INTO OFFENDER_NON_ASSOCIATIONS (OFFENDER_ID,NS_OFFENDER_ID,OFFENDER_BOOK_ID,NS_OFFENDER_BOOK_ID,NS_REASON_CODE, RECIP_NS_REASON_CODE)
-VALUES (-1001,-1002,-1,-2,'PER','PER');
+INSERT INTO OFFENDER_NON_ASSOCIATIONS (OFFENDER_ID,NS_OFFENDER_ID,OFFENDER_BOOK_ID,NS_OFFENDER_BOOK_ID,NS_REASON_CODE, RECIP_NS_REASON_CODE) VALUES (-1001,-1002,-1,-2,'PER','PER');
+INSERT INTO OFFENDER_NON_ASSOCIATIONS (OFFENDER_ID,NS_OFFENDER_ID,OFFENDER_BOOK_ID,NS_OFFENDER_BOOK_ID,NS_REASON_CODE, RECIP_NS_REASON_CODE) VALUES (-1002,-1001,-2,-1,'PER','VIC');
```

Comment:
Additional test data to reflect the inverse link so in this case perpetrator to victim.